### ISSUE TITLE
[EN][NPC][Dragonblight]Nishera the Garden Keeper

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4198]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Nishera should be patroling around the path at Emeral Dragonshrine . She does not.
**Expected behaviour**: Tell us what should happen instead
She should walk on the path around the Emeral Dragonshrine.
**Steps to reproduce the problem**:

1. Go to Emerald Dragonshrine 
2. Locate her.
3. Not walking.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[WoWhead Link to NPC](http://www.wowhead.com/npc=27255/nishera-the-garden-keeper)
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
path can be seen here http://www.wowhead.com/npc=27255/nishera-the-garden-keeper#map

**Comment by Rushor** [comment]
DB/Creature: Nishera the Garden Keeper - Add Pathing for Dragonblight zone

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4198
* commit: 5be264fedd61c4fe4efb726c07bdf8f79ab6e9f1

![GitHub Logo](http://i.imgur.com/FQ29Hv0.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
